lib/diaspora/shareable.rb

```diff
@@ -50,6 +50,14 @@ def subscribers
       end
     end
 
+    # Remote pods which are known to be subscribed to the post. Must include all pods which received the post in the
+    # past.
+    #
+    # @return [Array<String>] The list of pods' URIs
+    def subscribed_pods_uris
+      subscribers.map {|subscriber| subscriber.pod.try(:url_to, "") }.compact.uniq
```

Comment:
You're right. But then `subscribers.select(&:remote?).map(&:url).uniq` would work. I also thought about getting all unique `pod_id`s and then get all pods and only build the url once per pod.